src/App.js

```diff
@@ -1,11 +1,38 @@
-import Calculator from './components/Calculator';
+import {
+  Routes, Route, Link,
+} from 'react-router-dom';
+// import Calculator from './components/Calculator';
```

Comment:
- [ ] @otmanTR great work on your component , kindly remove this commented line, its not good practise having commented code in your codebase.